get/install.bash

```diff
@@ -2,37 +2,74 @@
 # webexec installation script
 #
 # This script is meant for quick & easy install via:
+#
+#   $ curl -L https://get.webexec.sh -o get-webexec.sh | bash get-webexec.sh
 set -x
-
-#   $ curl -L https://get.webexec.sh | bash
 SCRIPT_COMMIT_SHA=UNKNOWN
 LATEST_VERSION="0.15.3"
 
 # The latest release is currently hard-coded.
-echo "Installing " $LATEST_VERSION "version"
+echo "Installing webexec version " $LATEST_VERSION
          
-ARCH="$(uname -m)"  # -i is only linux, -m is linux and apple
-if [[ "$ARCH" = x86_64* ]]; then
+ARCH="$(uname -m | tr [:upper:] [:lower:])" 
+if [ "$ARCH" = x86_64* ]; then
     if [[ "$(uname -a)" = *ARM64* ]]; then
         ARCH='arm64'
     else
         ARCH="amd64"
     fi
-elif [[ "$ARCH" = i*86 ]]; then
+elif [ "$ARCH" = i386 ]; then
     ARCH='386'
 elif [[ "$ARCH" = armv6* ]]; then
     ARCH='armv6'
 elif [[ "$ARCH" = armv7* ]]; then
     ARCH='armv7'
-elif [[ "$ARCH" = aARCH64 ]]; then
+elif [[ "$ARCH" = aarch64 ]]; then
```

Comment:
Are you only providing a 32-bit ARM executable?

Reply:
My desktop is a mac mini M1 and it runs on 64bit there. 

TBH, I've lost track of the hw. architecture these last few year. I use goreleaser and copied the [config](https://github.com/tuzig/webexec/blob/master/.goreleaser.yml) from a project I liked. please take a look and see if there's anything missing.